src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -285,10 +291,23 @@ namespace winrt::TerminalApp::implementation
     // - true if the ApplicationState should be used.
     bool TerminalPage::ShouldUsePersistedLayout(CascadiaSettings& settings) const
     {
-        // If the setting is enabled, and we are the only window.
+        // GH#5000 Until there is a separate state file for elevated sessions we should just not
+        // save at all while in an elevated window.
         return Feature_PersistedWindowLayout::IsEnabled() &&
-               settings.GlobalSettings().FirstWindowPreference() == FirstWindowPreference::PersistedWindowLayout &&
-               _numOpenWindows == 1;
+               !_IsElevated() &&
+               settings.GlobalSettings().FirstWindowPreference() == FirstWindowPreference::PersistedWindowLayout;
+    }
```

Comment:
This could be static.

Reply:
Well, right now it isn't, but I know @zadjii-msft will have a convenient static IsElevated in the application state PR, and also might just obviate the neew for `_IsElevated` to be called here at all making this completely static.